sql/core/src/main/scala/org/apache/spark/sql/execution/commands/ddl.scala

```diff
@@ -0,0 +1,189 @@
+/*
```

Comment:
and then maybe for other things like `ExplainCommand` or `ClearCacheCommand` I would just leave them in `commands.scala`